Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -184,19 +190,22 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
     }
 
     private func setKeyValidity(_ interval: TimeInterval?) {
-        guard keyValidity != interval,
-            let interval = interval else {
```

Comment:
A `nil` interval wasn't meant to bail out... it's meant to be a request to reset the key validity interval.